# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/DCLayerTree.cpp
# Commit: 476bf3a723b1
# Full Hash: 476bf3a723b14b30f682b928d3375d3ec79f08c5
# Author: Sotaro Ikeda <sotaro.ikeda.g@gmail.com>
# Date: 2025-06-10 15:51:40
# Regressor Bug: 1945683
# File Overlap Count: 1
# Description:
#   Bug 1945683 - Enable layer compositor on Windows in Nightly r=gfx-reviewers,lsalzman
#   
#   All known rendering bugs of layer compositor have been addressed. Then is seems OK to remove nightly only limitation of pref gfx.webrender.layer-compositor.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251788
# ==============================================================================

diff -r 79051e9448c6 -r 476bf3a723b1 gfx/webrender_bindings/DCLayerTree.cpp
--- a/gfx/webrender_bindings/DCLayerTree.cpp	Tue Jun 10 11:14:17 2025 +0000
+++ b/gfx/webrender_bindings/DCLayerTree.cpp	Tue Jun 10 11:15:01 2025 +0000
@@ -547,13 +547,8 @@
 }
 
 bool DCLayerTree::UseLayerCompositor() const {
-// Only allow the layer compositor in nightly builds, for now.
-#ifdef NIGHTLY_BUILD
   return UseNativeCompositor() &&
          StaticPrefs::gfx_webrender_layer_compositor_AtStartup();
-#else
-  return false;
-#endif
 }
 
 void DCLayerTree::DisableNativeCompositor() {